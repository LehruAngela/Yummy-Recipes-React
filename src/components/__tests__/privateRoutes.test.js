import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import PrivateRoute from '../privateRoutes';

describe('PrivateRoute', () => {
    const wrapper = shallow(<PrivateRoute/>);
    const preventDefault = jest.fn();

    it('renders properly', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
        }); 
    

})