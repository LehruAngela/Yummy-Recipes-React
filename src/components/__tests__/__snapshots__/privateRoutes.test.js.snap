// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrivateRoute renders properly 1`] = `
<Route
  render={[Function]}
/>
`;
